[2024-07-07T12:07:17.481+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T12:07:17.570+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final_exam.starting manual__2024-07-07T12:07:10.168261+00:00 [queued]>
[2024-07-07T12:07:17.595+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final_exam.starting manual__2024-07-07T12:07:10.168261+00:00 [queued]>
[2024-07-07T12:07:17.597+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-07T12:07:17.640+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): starting> on 2024-07-07 12:07:10.168261+00:00
[2024-07-07T12:07:17.653+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=56) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-07T12:07:17.656+0000] {standard_task_runner.py:63} INFO - Started process 65 to run task
[2024-07-07T12:07:17.660+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final_exam', 'starting', 'manual__2024-07-07T12:07:10.168261+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpdv_cjfen']
[2024-07-07T12:07:17.662+0000] {standard_task_runner.py:91} INFO - Job 92: Subtask starting
[2024-07-07T12:07:17.764+0000] {task_command.py:426} INFO - Running <TaskInstance: Final_exam.starting manual__2024-07-07T12:07:10.168261+00:00 [running]> on host 257aaaf0c48f
[2024-07-07T12:07:18.044+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final_exam' AIRFLOW_CTX_TASK_ID='starting' AIRFLOW_CTX_EXECUTION_DATE='2024-07-07T12:07:10.168261+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-07T12:07:10.168261+00:00'
[2024-07-07T12:07:18.050+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T12:07:18.052+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-07T12:07:18.056+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Chuẩn bị cào dữ liệu"']
[2024-07-07T12:07:18.085+0000] {subprocess.py:86} INFO - Output:
[2024-07-07T12:07:18.093+0000] {subprocess.py:93} INFO - Chuẩn bị cào dữ liệu
[2024-07-07T12:07:18.097+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-07T12:07:18.100+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T12:07:18.163+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Final_exam, task_id=starting, run_id=manual__2024-07-07T12:07:10.168261+00:00, execution_date=20240707T120710, start_date=20240707T120717, end_date=20240707T120718
[2024-07-07T12:07:18.234+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T12:07:18.294+0000] {taskinstance.py:3498} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-07-07T12:07:18.298+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
